builder/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -ex
-source ./gpgcheck.sh | source builder/gpgcheck.sh
+source ./gpgcheck.sh || source builder/gpgcheck.sh
```

Comment:
I think this line should be only  `source builder/gpgcheck.sh`
Travis complains about `./gpgcheck.sh`